frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -350,70 +349,6 @@ export function ObjectDetailView({
   );
 }
 
-export function ObjectDetailWrapper({
```

Comment:
Moved to a different file - I think this might be good to do for each of the components that are in here. Maybe we can gradually do that and add tests for each component and its corresponding `unit.spec` file